docker/deployed/testnet/healthbot/config.json

```diff
@@ -6,41 +6,41 @@
         "Human": false,
         "Debug": true
     },
-    "Target": "https://testnet.tableland.network/rpc",
+    "Target": "https://testnets.tableland.network/rpc",
```

Comment:
Should we remove the`Target` entry in this config? (also in `staging`)

Reply:
Good point, will do.

Reply:
@brunocalza, done, added an extra commit. Ready to have another approval for merging.